GUIDE.md

```diff
@@ -0,0 +1,181 @@
+# Node Documentation
+
+Welcome to the Node.js documentation guide – or more colloquially, the docs on
+docs. If you're here, you're likely looking for guidance on how to write
```

Comment:
Ah! If you're using NY Times guide, I believe en dash rather than em dash is the way to go and (as you note) keep the spaces. So, basically, as you had it and ignore my dash comments. 

And, yeah, whatever you use, noting the standard your using will help settle bike shed debates quickly. (And nothing brings out the bike shedding in everyone like copyediting. I'm doing it here!)